libs/langchain/langchain/utilities/requests.py

```diff
@@ -58,12 +58,12 @@ async def _arequest(
         if not self.aiosession:
             async with aiohttp.ClientSession() as session:
                 async with session.request(
-                    method, url, headers=self.headers, auth=self.auth, **kwargs
```

Comment:
are you saying if auth is passed in as part of kwargs this leads to an error?

Reply:
Yes, if you see the aget it passes a auth parameter which gets populated into the kwargs and is sending another auth parameter here resulting in duplicate parameters. Aiohttp throws an error due to it.

Reply:
@baskaryan any update on this?

Reply:
any update?